mitodl/course-search-utils
# Fix the @mitodl/open-api-axios version

### What are the relevant tickets?





Quick fix.

### Description (What does it do?)


Sets the current compatible @mitodl/open-api-axios version in the package.json so we are not using `^` version ranges.

The current [validation](https://github.com/mitodl/course-search-utils/blob/main/src/hooks/validation.ts) code is compatible with `2024.9.16`, but `^2024.9.16` matches `2024.9.16` through to latest `2024.9.25`, which is is not compatible with.

Error with `2024.9.25` since removed `LearningFormatEnum`:
TypeError: Cannot convert undefined or null to object on 
https://github.com/mitodl/course-search-utils/blob/main/src/hooks/validation.ts#L66

### How can this be tested?


Install the branch into MIT Learn or any project. In package.json: 

```
"@mitodl/course-search-utils": "mitodl/course-search-utils.git#jk/fix-axios-api-version"
```

The project should build and run without type errors.
